Assignment2.yaml

```diff
@@ -0,0 +1,402 @@
+swagger: "2.0"
+info:
+  description: "Starwars API."
+  version: "1.0.0"
+  title: "SWAPI"
+host: "swapi.co"
+basePath: "/api/"
```

Comment:
extra forward / not required here.
This will make URLs like http://swapi.co/api//Vehicle